### ISSUE TITLE
Doc comments that include NON-ASCII characters not work as expected

### ISSUE THREAD

**Issue by magurotuna** [dtolnay/syn#772]
I prepared the following code:

```rust
fn main() {
    let s = r##"
/// Ä👍123こんにちは世界
#[doc = r"Ä👍123こんにちは世界"]
#[doc = "Ä👍123こんにちは世界"]
fn hoge() {}
"##;
    let t: syn::ItemFn = syn::parse_str(s).unwrap();
    dbg!(t);
}
```

(`こんにちは世界` means `Hello world` in English )

I expected that three doc comments in the above code were parsed in the same way, except for raw string literal.
But the result was as follows (extract only `attrs` section)

```
    attrs: [
        Attribute {
            pound_token: Pound,
            style: Outer,
            bracket_token: Bracket,
            path: Path {
                leading_colon: None,
                segments: [
                    PathSegment {
                        ident: Ident(
                            doc,
                        ),
                        arguments: None,
                    },
                ],
            },
            tokens: TokenStream [
                Punct {
                    op: '=',
                    spacing: Alone,
                },
                Literal {
                    lit: " \u{c4}\u{1f44d}123\u{3053}\u{3093}\u{306b}\u{3061}\u{306f}\u{4e16}\u{754c}",
                },
            ],
        },
        Attribute {
            pound_token: Pound,
            style: Outer,
            bracket_token: Bracket,
            path: Path {
                leading_colon: None,
                segments: [
                    PathSegment {
                        ident: Ident(
                            doc,
                        ),
                        arguments: None,
                    },
                ],
            },
            tokens: TokenStream [
                Punct {
                    op: '=',
                    spacing: Alone,
                },
                Literal {
                    lit: r"Ä👍123こんにちは世界",
                },
            ],
        },
        Attribute {
            pound_token: Pound,
            style: Outer,
            bracket_token: Bracket,
            path: Path {
                leading_colon: None,
                segments: [
                    PathSegment {
                        ident: Ident(
                            doc,
                        ),
                        arguments: None,
                    },
                ],
            },
            tokens: TokenStream [
                Punct {
                    op: '=',
                    spacing: Alone,
                },
                Literal {
                    lit: "Ä👍123こんにちは世界",
                },
            ],
        },
    ],
```

The first one, which is doc comment of three slashes style, seems different from the others.
Is this a bug of this crate?

Playground: https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=87894541210fed53beb979036f7bacaf

**Comment by dtolnay** [comment]
This was a proc-macro2 bug. Fixed in 1.0.9.

**Comment by magurotuna** [comment]
Awesome!
Thank you very much :)
